src/public/Interfaces.ts

```diff
@@ -95,6 +95,53 @@ export interface HttpRequest {
      * The HTTP request body as a UTF-8 string.
      */
     rawbody?: any;
+    /**
+     * Claims-based security information for the current HTTP request.
+     */
+    user?: ClaimsIdentity[];
+}
+
+/**
+ * An identity described by a collection of claims.
+ */
+export interface ClaimsIdentity {
+    /**
+     * The authentication type.
+     */
+    authenticationType: string | undefined;
+
+    /**
+     * The claim type that is used to determine which claims provide the name
+     * of this claims identity.
+     */
+    nameClaimType: string | undefined;
+
+    /**
+     * The claim type that is used to determine which claims provide the role
+     * of this claims identity.
+     */
+    roleClaimType: string | undefined;
+
+    /**
+     * The claims associated with this claims identity.
+     */
+    claims: Claim[];
+}
+
+/**
+ * A statement about a subject by an issuer. Represents an attribute of the
+ * subject that is part of an identity scenario.
+ */
+export interface Claim {
+    /**
+     * The value of the claim.
```

Comment:
Could this be more specific?

Reply:
@ConnorMcMahon what's your take?

Reply:
Perhaps a link to the RFC section on claims? https://tools.ietf.org/html/rfc7519#section-4